responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -148,6 +150,23 @@ def _get_classes(task_type, train, target_column, classes):
         else:
             return None
 
+    def _try_add_data_balance(self):
```

Comment:
This logic automatically adds data balance measures (`compute()` still has to be called) if the right conditions are met when user creates an `RAIInsights()` object.

@imatiach-msft @xuke444 @PIC123 Let me know what you think of this change and how we can re-shape it if it's too complicated 😊